src/Tooltip/index.tsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react'
+import {useSSRSafeId} from '@react-aria/ssr'
+import type {AnchorPosition, AnchorSide, AnchorAlignment} from '@primer/behaviors'
+import Box from '../Box'
+import {useAnchoredPosition} from '../hooks'
+import {SxProp, merge, BetterSystemStyleObject} from '../sx'
+
+type TooltipDirection = 'nw' | 'n' | 'ne' | 'e' | 'se' | 's' | 'sw' | 'w'
+type TooltipAlign = 'left' | 'right'
+
+export type TooltipProps = {
+  /** The text content of the tooltip. This should be brief and no longer than a sentence.
+   *  Marked as optional to support backward compatibility with aria-label. */
+  text?: string
+  /** @deprecated Use `text` instead */
+  'aria-label'?: string
```

Comment:
notes for reviewer: Marked `aria-label` as deprecated because it's not necessary it's always a label, it could be description.